docs/data-ai/data/visualize.md

```diff
@@ -35,42 +35,152 @@ See [configure a machine](/operate/get-started/supported-hardware/) for more inf
+   {{<imgproc src="/services/data/visualize-choose-location.png" resize="800x" style="width: 700px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a location for your workspace.">}}
 
-{{% /tablestep %}}
-{{% tablestep number=2 %}}
-**Add widgets**
+1. From the **Select machine** dropdown, select the machine that you would like to visualize data from.
 
-Click **Add widget** and add a **GPS** widget for any position-reporting sensor, a **stat** widget for current readings from a sensor, or a **time series** widget to graph data for any component that supports capturing numeric data.
-Use the widget header to configure the panel.
-Repeat as many times as necessary.
+   {{<imgproc src="/services/data/visualize-choose-machine.png" resize="800x" style="width: 700px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a machine for your workspace.">}}
 
-{{% /tablestep %}}
-{{% tablestep number=3 %}}
-**Select a machine**
+### Add a widget
 
-Now, select a machine with which to make your teleop workspace come to life.
-Click **Select machine** and select your configured machine.
+1. Click **Add widget** and select a widget type to create a new widget of that type on your workspace:
 
-Your dashboard now shows the configured widget for the data from your machine.
-For example, a time series graph measuring noise over time for a sensor component:
+   {{<imgproc src="/services/data/visualize-add-widget.png" resize="800x" style="width: 300px" class="fill imgzoom shadow" declaredimensions=true alt="Choosing a widget type from the widget menu.">}}
 
-{{< imgproc src="/services/data/time-series.png" alt="Time series widget measuring noise over time." style="width:500px" resize="1200x" class="imgzoom fill shadow" >}}
+1. To configure the widget, click the pencil icon in the top right of your widget:
 
-{{% /tablestep %}}
-{{< /table >}}
+   {{<imgproc src="/services/data/visualize-widget-configure.png" alt="Click the pencil icon to configure your widget." style="width:500px" resize="1200x" class="imgzoom fill shadow" >}}
+
+   {{% alert title="Tip" color="tip" %}}
+   Select a location and machine for your workspace to populate the widget configuration dropdowns using sensor names and data from that machine.
+   {{% /alert %}}
+
+
+1. To arrange widgets on your workspace, click and drag the grid icon in the top left of your widget:
+
+   {{<imgproc src="/services/data/visualize-widget-move.png" alt="Click the grid icon to move a widget." style="width:500px" resize="1200x" class="imgzoom fill shadow" >}}
+
+For example, the following image shows a time series graph measuring noise over time for a sensor component:
+
+{{<imgproc src="/services/data/visualize-widget-time-series.png" alt="Time series widget measuring noise over time." style="width:700px" resize="1200x" class="imgzoom fill shadow" >}}
```

Comment:
isn't this one redundant with widget types below which shows the time series widget?

Reply:
Looks like nothing has changed here since last review?